src/docs/create-mapping-commons.md

```diff
@@ -0,0 +1,47 @@
+# Getting started
+
+- First, create a virtual environment of your choice (anaconda, venv, poetry etc.).
```

Comment:
I think less technical users will need some guidance on how to create a virtual environment. However, with the numerous options available, I'm not sure whether it would be better to provide an example or 2 or to just include some reference links.

Reply:
Good call! I've updated the documentation with a 'best practices guide' that our group uses.